pkg/functions/function_test.go

```diff
@@ -398,3 +398,16 @@ func TestFunction_Stamp(t *testing.T) {
 		t.Fatal("expected journal log not found")
 	}
 }
+
+func TestFunction_LoadLocalSettings(t *testing.T) {
```

Comment:
So, now you can expand this test to `NewFunction` the test function, set `.LocalSettings` to `true`, then `.Write()` it.  Then `NewFunction` it again, and check that `.localSettings.Remote` is still true 👍🏻